examples/embeddable_examples/public/react_embeddables/saved_book/saved_book_react_embeddable.tsx

```diff
@@ -137,6 +139,10 @@ export const getSavedBookEmbeddableFactory = (core: CoreStart) => {
         }
       );
 
+      const showLibraryCallout =
+        apiHasParentApi(api) &&
+        typeof (api.parentApi as PresentationContainer)?.replacePanel === 'function';
```

Comment:
Why this check?

Reply:
without this check, the embeddable shows a callout stating whether its stored in library or not in library. Since the pageApi does not support library actions, this message is distracting from the example

Reply:
Why the `replacePanel` check specifically? 🤔 Is it just because the `pageApi` doesn't have it? I wonder if we could add a comment here to describe that

Reply:
`replacePanel` is taken from link and unlink action compatibility methods https://github.com/elastic/kibana/blob/main/src/plugins/dashboard/public/dashboard_actions/unlink_from_library_action.tsx#L49.

I can add a comment to make this more clear.